trunk/src/app/srs_app_http_stream.cpp

```diff
@@ -1032,13 +1056,40 @@ srs_error_t SrsHttpStreamServer::http_mount(SrsRequest* r)
 void SrsHttpStreamServer::http_unmount(SrsRequest* r)
 {
     std::string sid = r->get_stream_url();
-    
-    if (sflvs.find(sid) == sflvs.end()) {
+
+    std::map<std::string, SrsLiveEntry*>::iterator it = sflvs.find(sid);
```

Comment:
About the name of `sflvs` and `tflvs`, is there any short-terms that I didn't know?, it store `FLV/TS/MP3/AAC` streams, but name it as `http-live-stream`, short as `hls`, seems a conflict with apple `HLS`, is there any better ideas? or just a mis-concept in my mind.

Reply:
```
    // The http live streaming template, to create streams.
    std::map<std::string, SrsLiveEntry*> tflvs;
    // The http live streaming streams, crote by template.
    std::map<std::string, SrsLiveEntry*> sflvs;
```
Perhaps this comment can answer your question?

Reply:
Accept. Rename tflvs to templateHandlers, sflvs to streamHandlers.